src/constants.ts

```diff
@@ -2,6 +2,7 @@ export const LineType = {
   Added: 'AddedLine',
   Deleted: 'DeletedLine',
   Unchanged: 'UnchangedLine',
+  MissingEof: 'MissingEofLine',
```

Comment:
@zmully 
```suggestion
  MissingEof: 'MissingEofLine',
```

What about changing the value and name with `NoNewLineEOF` or `MissingNewLineEOF`?